core/dbt/cli/main.py

```diff
@@ -6,7 +6,10 @@
 from dbt.adapters.factory import adapter_management
 from dbt.cli import params as p
 from dbt.cli.flags import Flags
+from dbt.events.functions import setup_event_logger
```

Comment:
Why are we pulling in logging stuff?

Reply:
I think this is because I cherry picked your commit to be able to access those attributes in flags.